app/helpers/react_components/student/mentoring_session.rb

```diff
@@ -6,26 +6,44 @@ class MentoringSession < ReactComponent
       def to_s
         super(
           "student-mentoring-session",
-          {
-            user_handle: student.handle,
-            request: SerializeMentorSessionRequest.(request, student),
-            discussion: discussion ? SerializeMentorDiscussionForStudent.(discussion) : nil,
-            track: SerializeMentorSessionTrack.(track),
-            exercise: SerializeMentorSessionExercise.(exercise),
-            iterations:,
-            mentor: mentor_data,
-            track_objectives: user_track&.objectives.to_s,
-            out_of_date: solution.out_of_date?,
-            videos:,
-            links: {
-              exercise: Exercism::Routes.track_exercise_mentor_discussions_url(track, exercise),
-              create_mentor_request: Exercism::Routes.api_solution_mentor_requests_path(solution.uuid),
-              learn_more_about_private_mentoring: Exercism::Routes.doc_path(:using, "feedback/private"),
-              private_mentoring: solution.external_mentoring_request_url,
-              mentoring_guide: Exercism::Routes.doc_path(:using, "feedback/guide-to-being-mentored")
+          to_h
+        )
+      end
+
+      def to_h
+        {
+          user_handle: student.handle,
+          request: SerializeMentorSessionRequest.(request, student),
+          discussion: discussion ? SerializeMentorDiscussionForStudent.(discussion) : nil,
+          track: SerializeMentorSessionTrack.(track),
+          exercise: SerializeMentorSessionExercise.(exercise),
+          iterations:,
+          mentor: mentor_data,
+          track_objectives: user_track&.objectives.to_s,
+          out_of_date: solution.out_of_date?,
+          videos:,
+          donation: {
+            user_signed_in: true,
+            captcha_required: false,
+            recaptcha_site_key: Exercism.secrets.recaptcha_site_key,
```

Comment:
Do we need this, @dem4ron as `captcha_required` is always false?

Reply:
then probably we don't

Reply:
OK. I'll leave you to remove it! :)